recipes/timu-spacegrey-theme

```diff
@@ -0,0 +1,3 @@
+(timu-spacegrey-theme :repo "aimebertrand/timu-spacegrey-theme"
+                      :fetcher gitlab
+                      :files ("README.org" "timu-spacegrey-theme.el" "timu-spacegrey-theme.png"))
```

Comment:
```suggestion
(timu-spacegrey-theme :repo "aimebertrand/timu-spacegrey-theme")
```
The default recipe should work here -- we usually don't package READMEs and screenshots.